pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -336,7 +336,7 @@ def newpkgcb(pkgId, name, arch):
                     downloader = self.remote.get_downloader(url=updateinfo_url)
                     downloaders.append([downloader.run()])
                 elif record.type in MODULAR_REPODATA:
-                    modules_url = urljoin(self.remote.url, record.location_href)
+                    modules_url = urljoin(remote_url, record.location_href)
```

Comment:
Changes in this file also seem to be unrelated to the performance tests. Or are they?

Reply:
these changes were made because performance tests were breaking for multi-variants

Reply:
done: https://github.com/pulp/pulp_rpm/pull/1475